spec/support/factory_bot.rb

```diff
@@ -7,4 +7,10 @@
     password_confirmation { 'password' }
   end
 
+  factory :game do
```

Comment:
you may want to consider separate factory bot files for each type of factory - in the `spec/support` folder you can create a `factories` folder and name your factories `users.rb`, `games.rb`